src/device-api/v2.ts

```diff
@@ -3,7 +3,7 @@ import { NextFunction, Request, Response, Router } from 'express';
 import * as _ from 'lodash';
 import { fs } from 'mz';
 
-import { ApplicationManager } from '../application-manager';
+import ApplicationManager = require('../application-manager');
```

Comment:
Because it was wrong before this. The coffeescript module does `exports = class ApplicationManager`, but the typings didn't reflect this (and I actually hit an error when I tried to import and instantiate the class in this module).

Reply:
Right, that explains that. :)